RecipeApp/Modules/RecipeBook/View/RecipeBookViewController.swift

```diff
@@ -0,0 +1,84 @@
+//
+//  RecipeBookViewController.swift
+//  RecipeApp
+//
+//  Created by Hadley Igoe on 2019-12-17.
+//  Copyright © 2019 Hadley Igoe. All rights reserved.
+//
+
+import UIKit
+
+final class RecipeBookViewController: UIViewController, ViewControllerCreatableWithViewModel {
+    
+    @IBOutlet weak var tableView: UITableView!
+    @IBOutlet weak var emptyView: UIView!
+    
+    var viewModel: RecipeBookViewModelProtocol!
+    weak var coordinator: RecipeBookCoordinator?
```

Comment:
This isn't used so remove it. And don't forget that you wouldn't use it anyway since you're using MVVM.